build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,109 @@
+# FeatureGates is a map of feature names to bools that enable or disable experimental features.
```

Comment:
Why duplicate feature gates here? They should just be in antrea-agent.conf.

Reply:
yes that is right. @adwaitni removed it now. We initially thought maintain separate feature flag definition sets for podwatch needs. But that doesn't make sense in the broader picture.